ngx_http_sxg_filter_module.c

```diff
@@ -687,25 +687,32 @@ static ngx_int_t ngx_http_sxg_filter_init(ngx_conf_t* cf) {
     }
 
     if (!is_valid_config(cf, nscf)) {
-      ngx_log_error(NGX_LOG_NOTICE, cf->log, 0,
+      ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
```

Comment:
nginx log levels appears to correspond to syslog.  Does panic or error better capture your intent?

https://en.wikipedia.org/wiki/Syslog#Severity_level

Reply:
In nginx convention, misconfigured config file is alerted with `EMERG`.

```2020/05/12 05:11:48 [emerg] 19#19: unknown directive "foobar" in /etc/nginx/sites-enabled/nginx-sxg.conf```